sdk/storage/Azure.Storage.Blobs/src/Generated/BlobRestClient.cs

```diff
@@ -15988,15 +15988,15 @@ public static partial class BlobsModelFactory
         /// Creates a new AccountInfo instance for mocking.
         /// </summary>
         public static AccountInfo AccountInfo(
-            Azure.Storage.Blobs.Models.SkuName skuName,
+            bool isHierarchicalNamespaceEnabled,
```

Comment:
Something has gone wrong, this file should not have been modified. We should probably do an interactive rebase and remove this change or recreate the branch